source/designers-guide/custom-detail-page/index.md

```diff
@@ -486,3 +486,17 @@ The code to remove the image zoom is pretty simple. We check if the main `conten
 If you want to take a closer look at all the code we have written in this tutorial you can download this plugin:
 
 <strong><a href="{{ site.url }}/exampleplugins/SwagCustomDetailTheme.zip">Plugin download</a></strong>
+
+The Theme is packed inside a plugin, as that's a requirement to distribute themes via our store.
+The structure of the Theme is exactly the same as before, it just sits a few layers deeper in the directory hierarchy:
```

Comment:
The structure of the theme is exactly the same as before, it is just placed a few layers deeper in the directory hierarchy:

Reply:
Done